UnityMcpBridge/UnityMcpServer~/src/tools/manage_script.py

```
@@ -1,29 +1,384 @@
+            normalized_edits = []
+            for e in edits or []:
+                e2 = dict(e)
+                has_all = all(k in e2 for k in ("startLine","startCol","endLine","endCol"))
+                if has_all:
+                    zero_based = False
+                    for k in ("startLine","startCol","endLine","endCol"):
+                        try:
+                            if int(e2.get(k, 1)) < 1:
+                                zero_based = True
+                        except Exception:
+                            pass
```

Ruff (0.12.2)

`try`-`except`-`pass` detected, consider logging the exception

(S110)